packages/react-examples/src/react-menu/react-menu.stories.scss

```diff
@@ -0,0 +1 @@
+/* Storybook SCSS file for package: react-menu */
```

Comment:
Do we need this file?

Reply:
removed both scss files and replaced with index.ts placeholder